Tests/AsyncHTTPClientTests/HTTPClientTests.swift

```diff
@@ -1701,7 +1701,7 @@ class HTTPClientTests: XCTestCase {
             XCTAssertNoThrow(try httpClient.syncShutdown())
         }
         // create random block of text
-        let uint8Array = Array<Int8>(repeating: 0, count: 16384).map { _ in Int8.random(in: 32...127)}
+        let uint8Array = [Int8](repeating: 0, count: 16384).map { _ in Int8.random(in: 32...127)}
         let asciiString = String(utf8String: uint8Array)!
```

Comment:
isn't something like this nicer to read?

```
String(decoding: Array(repeating: UInt8(0), count: 16).map { _ in (UInt8(ascii: "A")...UInt8(ascii: "z")).randomElement()! }, as: UTF8.self)
```

`String(decoding:as:)` also doesn't return optional and is the fastest initialiser because it's in the stdlib and not Foundation (ie. doesn't go through NSString).